lomap/dbmol.py

```diff
@@ -532,7 +535,6 @@ def compute_mtx(self, a, b, strict_mtx, loose_mtx, true_strict_mtx, MCS_map):
                             max3d=self.options['max3d'],
                             element_change=self.options['element_change'])
                         ml = MC.all_atom_match_list()
-                        self.set_MCSmap(i, j, ml)
```

Comment:
removing this line will probably break something (`get_MCSmap`)

Reply:
No, this line is useless.
- In single thread mode, `get_MCSmap` is called later at [dbmol.py#L590](https://github.com/OpenFreeEnergy/Lomap/blob/v2.1.0/lomap/dbmol.py#L590).
- In multi-process mode, `get_MCSmap` is called at [dbmol.py#L645](https://github.com/OpenFreeEnergy/Lomap/blob/v2.1.0/lomap/dbmol.py#L645). Calling `set_MCSmap` in a child process will only update the copy of `self.mcs_map_store` rather than that of the parent process.